File: 2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.py
# Made by Piyush
def main():
    testCases = int(input())
    for _ in range(testCases):
        l = int(input())
        arr = list(map(int, input().split()))
        print(arr.index(min(arr)) + 1)

if __name__ == '__main__':
    main()

File: 2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.c
//Made by Divyesh Puri
#include <stdio.h>
int main(){
	int t;
	scanf("%d", &t);
	while (t--){
		int n, i, k = 100000, index;
		scanf("%d", &n);
		int arr[n];
		for (i = 0; i < n; i++){
			scanf("%d", &arr[i]);
			if (arr[i] < k){
				k = arr[i];
				index = i;
			}
		}
		printf("%d\n",index + 1);
	}
}
